src/main/java/net/neoforged/neoforge/registries/datamaps/builtin/NeoForgeDataMaps.java

```diff
@@ -38,6 +39,18 @@
  * synced so that mods can use them on the client side.
  */
 public class NeoForgeDataMaps {
+    /**
+     * The {@linkplain EntityType} data map that replaces {@link VillagerHostilesSensor#ACCEPTABLE_DISTANCE_FROM_HOSTILES}.
+     * <p>
+     * The location of this data map is {@code neoforge/data_maps/item/compostables.json}, and the values are objects with 1 field:
+     * <ul>
+     * <li>{@code chance}, a float between 0 and 1 (inclusive) - the chance that the item will add levels to the composter when composted</li>
+     * </ul>
+     *
+     * The use of a float as the value is also possible, though discouraged in case more options are added in the future.
+     */
```

Comment:
Your documentation here is wrong, copy-pasted from a different datamap doc (compostables). Update to refer to the actual typing (EntityType) and relevant class references.